### ISSUE TITLE
8 GPU - Last Enable/Disabled Card Low Hashrate

### ISSUE THREAD

**Issue by F1FTW3D** [fireice-uk/xmr-stak#1107]
Hi,

I'm having an odd issue with an 8-GPU Vega mining rig where the last GPU I disable/re-enable for AMD blockchain driver purposes experiences a low hash rate.

Detail:
-To work with the AMD blockchain driver, I disable all GPUs and re-enable in Device Manager
-All GPUs show 800+ h/s per thread, except the last one I disable/re-enable
-Tried with stock settings, numerous restarts, etc.
-Whichever GPU I re-enable last is the slower one (600 h/s per thread)

XMR Stak Settings:
-2x threads per card (0-7)
-1932 intensity
-8 worksize
-false affine_to_cpu
-true strided_index

Rig:
XMR Stak 2.2.0 c4400d19
Windows 10, 8 GB Ram
Asus B250 Mining Expert Motherboard
72 GB Page File
6 - MSI Air Boost Vega 56
1 - XFX Vega 56 Dual
1 - XFX Vega 64

Thank you in advance!

**Comment by eliashdezr** [comment]
Redo this guide as many times necessary to have your Vegas working as expected:
http://vega.miningguides.com/

You are doing something wrong.

I repeat, do it AS MANY TIMES NECESSARY.

**Comment by F1FTW3D** [comment]
That guide was actually one of the ones I used to set up the rig initially. The good news is I got it working, and it was one of two things:

A) I didn't have the EnableUlps and EnableCrossFireAutoLink registry settings set to 0, though another rig worked just fine without doing this
B) I was using a physically connected monitor but started only using remote desktop

It's still a bit finicky, with any change in OC settings causing it to drop to ~600/thread but now it bounces back after a few seconds.

Also, just FYI I didn't use the soft power tables or change the voltages for any cards. For my rig, changing the power target accomplished the same thing, verified with watt/amp meters.
